ios/RNSScreen.mm

```diff
@@ -622,6 +670,9 @@ - (void)updateProps:(facebook::react::Props::Shared const &)props
     [self setStackPresentation:newStackPresentation];
   }
 
+  // This must be called after setter for stackPresentation
+  [self updatePresentationStyle];
```

Comment:
This it the case, because retrieving `sheetPresentationController` from view controller requires `modalPresentationStyle` to be set to one of [UIModalPresentationPageSheet](https://developer.apple.com/documentation/uikit/uimodalpresentationstyle/uimodalpresentationpagesheet?language=objc) or [UIModalPresentationFormSheet](https://developer.apple.com/documentation/uikit/uimodalpresentationstyle/uimodalpresentationformsheet?language=objc). We do not support [UIModalPresentationPageSheet](https://developer.apple.com/documentation/uikit/uimodalpresentationstyle/uimodalpresentationpagesheet?language=objc), nevertheless this is done inside `stackPresentation` setter.

On Paper this effect is achieved by calling this method in `didSetProps:` method.